dynamo/preprocessing/utils.py

```diff
@@ -1145,19 +1152,19 @@ def scale(
         The scaled AnnData object.
     """
 
-    layers = DynamoAdataKeyManager.get_available_layer_keys(adata, layers)
-    has_splicing, has_labeling, _ = detect_experiment_datatype(adata)
-
-    if scale_to_layer is None:
-        scale_to_layer = "total" if has_labeling else None
-        scale = scale_to / adata.layers[scale_to_layer].sum(1)
-    else:
-        scale = None
+    # if layers is None:
+    #     layers = DynamoAdataKeyManager.get_available_layer_keys(adata, layers="all")
+    # has_splicing, has_labeling = detect_experiment_datatype(adata)[:2]
+    #
+    # if scale_to_layer is None:
+    #     scale_to_layer = "total" if has_labeling else None
+    #     scale = scale_to / adata.layers[scale_to_layer].sum(1)
+    # else:
+    #     scale = None
 
```

Comment:
why this is deleted? the above code will identify layers automatically when layers is None based on the DynamoAdataKeyManager.get_available_layer_keys function

Reply:
Retrieved